catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/SubscriptionsPersistentStoreImpl.java

```diff
@@ -60,7 +60,7 @@ public SubscriptionsPersistentStoreImpl(PersistentStore persistentStore) {
     LOCK.lock();
     try {
       List<Map<String, Object>> results =
-          persistentStore.get(SubscriptionsPersistentStore.SUBSCRIPTIONS_TYPE, q);
+          persistentStore.get(SubscriptionsPersistentStore.SUBSCRIPTIONS_TYPE, q, 0, 1000);
```

Comment:
✏️ Create a variable for 1000 (`MAX_PAGE_SIZE`) and for 0 (`DEFAULT_START_INDEX`)

Reply:
Added dependency on `PersistentStoreImpl.java` and used the constants defined there.